test/org/sagebionetworks/bridge/play/controllers/ParticipantControllerTest.java

```diff
@@ -159,17 +159,30 @@ public void oddParametersUseDefaults() throws Exception {
 
     @Test
     public void getParticipant() throws Exception {
-        StudyParticipant studyParticipant = new StudyParticipant.Builder().withFirstName("Test").build();
-        
-        when(participantService.getParticipant(STUDY, CALLER_ROLES, ID)).thenReturn(studyParticipant);
+        STUDY.setHealthCodeExportEnabled(true);
```

Comment:
Modifying a static member will persist that change across tests. This is bad. If you need to modify the study, create a new study for each test instead of modifying the old one.